### ISSUE TITLE
Simpler setup on windows

### ISSUE THREAD

**Issue by rishavs** [oprypin/crsfml#36]
I was wondering i there is a simpler way to setup crsfml on windows. The current process is fairly complex and is likely to be a huge hurdle to people thinking of trying it out.

IMO, a great solution will be to have it as a scoop install where users can just do `scoop install crsfml` and that fetches the binaries, adds the env vars and creates all the folders needed.

**Comment by oprypin** [comment]
The difficulty isn't on this side mostly, it's on Crystal, and people there don't seem very interested in simplifying this even in trivial ways.

The scary thing to me is automating the download and installation of VS Build Tools. If there are automated solutions, sure, I'd be happy to hear about them. Not familiar with `scoop` or anything else "modern" on Windows.
Also needs cmake btw.

Anyway, funnily enough, setting up GitHub Actions CI is very very easy for this.

**Comment by rishavs** [comment]
Agreed. the focus of Crystal maintainers have been puzzling for a while.

regarding the flow; from what I understand, the CI will generate the Windows binaries.
Package managers like Scoop/Winget/Chocolatey will then download the latest binary, install other dependencies, if any, add the env vars and create the folders. After that, whenever new binaries releases are generated in the CI, the package manager can fetch it with an update commend.

I have also added an issue in the Sccop repo. Hopefully, it will be  picked up. Closing this item as there doesn't seems to be anything immediate to do here.
https://github.com/lukesampson/scoop/issues/4114
